scalding-core/src/main/scala/com/twitter/scalding/typed/Grouped.scala

```diff
@@ -195,14 +199,14 @@ sealed trait CoGrouped[K, +R] extends KeyedListLike[K, R, CoGrouped]
    * but it is not clear how to generalize that for general cogrouping functions.
    * For now, just do a normal take.
    */
-  override def bufferedTake(n: Int): CoGrouped[K, R] =
+  override def bufferedTake(n: Int)(implicit q: Quoted): CoGrouped[K, R] =
```

Comment:
I'd rather not take an implicit here if it can't contribute to row/column filtering, which I can't see how it can in the case of take.